### ISSUE TITLE
refactor out importants

### ISSUE THREAD

**Issue by bnfish** [asascience/oceansmap-landing#4]
https://github.com/asascience/oceansmap-landing/pull/3#discussion_r81990569

**Comment by lukecampbell** [comment]
Yes, thank you. I really wish CSS didn't support `important!`.  It makes a project with lots of pages and lots of style rules to become completely unmaintainable when they are used. There's very little reason to use `important!` when a proper and more accurate selector will work.

More importantly, __(pun maybe intended)__ if we need to style a specific area of the page, if `important!` is used anywhere in any parent selector, all subsequent styling rules are overridden. And then the developer has to debug and work backwards to find why. And once the `important!` statement is found. That developer will need to refactor potentially a lot of CSS to properly make the change.
